links/index.html

```diff
@@ -53,7 +53,7 @@ <h2>Links</h2><p>This page will be extended with interesting links for all WebSV
 
 <h3>Old Links Section</h3>
 
-<ul><li><a href="http://websvn.tigris.org/">Tigris.org project page</a></li><li><a href="http://www.howtoforge.com/debian_subversion_websvn">Setting up subversion and websvn on debian</a></li><li><a href="https://www.ohloh.net/p/websvn">Analysis of WebSVN source on Ohloh.net<br /></a></li></ul><p>&nbsp;</p></div>
+<ul><li><a href="http://websvn.tigris.org/">Tigris.org project page</a></li><li><a href="http://www.howtoforge.com/debian_subversion_websvn">Setting up subversion and websvn on debian</a></li><li><a href="https://www.openhub.net/p/websvn">Analysis of WebSVN source on Ohloh.net<br /></a></li></ul><p>&nbsp;</p></div>
```

Comment:
The text for the link should be changed as well:

> Analysis of WebSVN source on Ohloh.net

vs.

> Analysis of WebSVN source on OpenHub.net

Reply:
Done.